app/models/form/question.rb

```diff
@@ -110,12 +110,16 @@ def displayed_answer_options(log, _current_user = nil)
   end
 
   def action_text(log)
+    displayed_as_answered?(log) ? "Change" : "Answer"
+  end
+
+  def displayed_as_answered?(log)
```

Comment:
This is maybe a bit cowardly - this logic (taken from the forms helper originally) is a bit different from `completed?` below so I left it intact, but I'm not clear on why.